lapce-ui/src/find.rs

```diff
@@ -205,6 +234,45 @@ impl Widget<LapceTabData> for FindBox {
         );
         self.input.paint(ctx, data, env);
 
+        let mut index = None;
+        let cursor_offset = buffer.editor.cursor.offset();
+
+        for i in 0..buffer.doc.find.borrow().occurrences().regions().len() {
+            let region = buffer.doc.find.borrow().occurrences().regions()[i];
+            if region.min() <= cursor_offset && cursor_offset <= region.max() {
+                index = Some(i);
+            }
+        }
+
+        let text_layout = ctx
+            .text()
+            .new_text_layout(if buffer.doc.find.borrow().occurrences().len() > 0 {
+                format!(
+                    "{}/{}",
+                    match index {
+                        Some(index) => format!("{}", index + 1),
+                        None => "?".to_string(),
```

Comment:
It looks a bit odd like ```?/215```, maybe we just show ``` 215 found ``` instead?

Reply:
Hmmm yeah agreed, that would make more sense.

Reply:
Now it shows `215 results` instead of `?/215`, so I presume this is resolved.